src/components/custom/_dhero.js

```diff
@@ -128,14 +128,14 @@ const DHero = ({
                     <div>
                         <LinkWrapper>
                             {join_us_for_free ? (
-                                <TryForFree secondary>
+                                <TryForFree secondary="true">
```

Comment:
no need to pass in `="true"`, just secondary is enough since its expecting boolean

Reply:
but @oskar-binary said to include it to avoid the warning in console :D so which one is right?

Reply:
well.. in that case its okay, can skip it 👍 the solution for those console error is here https://styled-components.com/docs/faqs#why-am-i-getting-html-attribute-warnings . adding ="true" wont be avoid all the warning console actually. we will do that in separate card instead